params.go

```diff
@@ -61,6 +61,20 @@ func (f *RequestValues) Set(key, val string) {
 	f.Add(key, val)
 }
 
+// Get retrieves the list of values for the given key.  If no values exist
+// for the key, an empty list will be returned.
+//
+// Note that Get is O(n) and may be quite slow for a very large parameter list.
+func (f *RequestValues) Get(key string) []string {
+	results := []string{}
```

Comment:
Fairly minor, but can we return `nil` instead of an empty list when `Get` doesn't find anything?

You should be able to do this by just changing this line to:

``` go
var results []string
```

`append` will automatically take care of list initialization for you.

(Also, the doc string will need an update too. Thanks!)